docs/api-reference/attribute-manager.md

```diff
@@ -1,6 +1,6 @@
 # AttributeManager Class
 
-The `AttributeManager` class automated attribute allocation and updates.
+The [`AttributeManager`](https://github.com/uber/deck.gl/blob/master/src/lib/attribute-manager.js) class provides automated attribute allocations and updates.
```

Comment:
Nit: When done this way, it is not clear what is a doc link and what is a source link.

I thought having a link that actually says `Source` (perhaps at the bottom of the page) would be cleaner. See for instance how THREE.js does it https://threejs.org/docs/api/math/Box2.html